build/build.py

```diff
@@ -267,10 +266,6 @@ def main():
       "--cuda_path",
       default="/usr/local/cuda",
       help="Path to the CUDA toolkit.")
-  parser.add_argument(
-      "--cudnn_path",
-      default="/usr/local/cuda",
```

Comment:
Maybe keep the flag but change the default to None or ""? Then people can still manually override if TF can't find it.

Reply:
The issue was that I was unable to find *any* setting of the flag that worked. And the value that it sets is marked as deprecated in the TF config anyway.